src/uniprotkb/components/entry/__tests__/__snapshots__/EntryMain.spec.tsx.snap

```diff
@@ -821,15 +821,6 @@ exports[`Entry view should render 1`] = `
             <h3>
               GO Annotations
             </h3>
-            <div
-              class="GoRibbon__container"
-            >
-              <div
-                class="GoRibbon__container__message"
-              >
-                Loading...
```

Comment:
Just wondering where this went to? I can see the `Loading...` message in the browser.

Reply:
If this is what I think we don't want to test the GO ribbon component itself so I mocked it (returns an empty object)

Reply:
Ah that's much nicer.

Reply:
We were getting warnings in the results because of it! 😿

Reply:
I don't know why I put a crying cat there. Still adjusting to the usage of emojis.